registry.yaml

```diff
@@ -795,6 +795,22 @@ packages:
   format_overrides:
   - goos: darwin
     format: zip
+- type: github_release
+  repo_owner: Peltoche
+  repo_name: lsd
+  asset: 'lsd-{{.Version}}-{{.Arch}}-{{.OS}}.{{.Format}}'
+  link: https://github.com/Peltoche/lsd
```

Comment:
```suggestion
```

You can omit this because it is equivalent to the default value.
https://github.com/suzuki-shunsuke/aqua/blob/main/docs/registry_config.md#default-values-of-github_release-package